experiment/src/data_loader.py

```diff
@@ -121,7 +125,9 @@ def eval_no_model(df: pd.DataFrame, df_missing: pd.DataFrame):
 
 
 def eval_with_model(df: pd.DataFrame, df_missing: pd.DataFrame, predictions: np.ndarray):
-    df["Correct"] = df["GroundTruth"] == predictions
+    # df["Correct"] = df["GroundTruth"] == predictions
```

Comment:
If you want to remain this line.. I think it would be good to add more comment next or below this line.